fusor-ember-cli/app/controllers/subscriptions/management-application.js

```diff
@@ -36,8 +36,6 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, {
       this.get('sessionPortal').save();
       this.set('upstreamConsumerUuid', managementApp.get('id'));
       this.set('upstreamConsumerName', managementApp.get('name'));
-      // show selected UUID in url
-      return this.transitionTo('subscriptions.management-application.consumer', managementApp.get('id'));
```

Comment:
We were using this to display the consumer_id in the URL, but we weren't showing any details in the template.  Since we trigger on willTransition, it would cause an update on click.  I didn't see any harm in removing it, but let me know and we can discuss it.